view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js

```diff
@@ -134,7 +134,7 @@ define(
 
                 let componentConfig = {
                     enableStoreDetails: self.getEnableStoreDetails(),
-                    brands: self.getAvailableCardTypeAltCodes(),
+                    brands: self.paymentMethodsResponse.getPaymentMethods(),
```

Comment:
This is probably wrong. `brands` should be filled with `paymentMethods`'s `brands` array. Could you please dump the outputs of this `self.getAvailableCardTypeAltCodes()` and this `self.paymentMethodsResponse.getPaymentMethods()` and compare the results?